src/pick.js

```diff
@@ -9,7 +8,7 @@ var _indexOf = require('./internal/_indexOf');
  * @func
  * @memberOf R
  * @category Object
- * @sig [String] -> {String: *} -> {String: *}
+ * @sig [k] -> {k: v} -> {k: v}
```

Comment:
`String` seems more appropriate than `k`. The latter suggests the list can contain values of any type.

Also, `{k: v}` suggests all the values in the object must be of the same type, which needn't be the case.